editor.planx.uk/src/@planx/components/shared/BaseOptionsEditor.tsx

```diff
@@ -84,7 +84,7 @@ export const BaseOptionsEditor: React.FC<BaseOptionsEditorProps> = (props) => (
             ...props.value,
             data: {
               ...props.value.data,
-              val: targetValue ? targetValue : undefined,
+              val: targetValue ?? "",
```

Comment:
Here's the issue - the input it originally `undefined` and then gets assigned to a string (or `null`) on change which toggles between controlled/uncontrolled.